### PR TITLE
add danielgindi-jquery-backstretch v2.1.12

### LINKED ISSUE #7296: [Request] Add jquery.backstretch
**Library name:** jquery.backstretch
**Git repository url:** https://github.com/danielgindi/jquery-backstretch/
**npm package url(optional):** https://www.npmjs.com/package/jquery.backstretch
**License(s):** MIT
**Official homepage:** 
**Wanna say something? Leave message here:**

The original jquery.backstretch repository is practically dead, and this is a fork that had much evolution.

Now there are requests to CDN it... https://github.com/danielgindi/jquery-backstretch/issues/19

### PR DESCRIPTION
**Pull request by PeterDaveHello**
** This PR is a live demo for @cdnjs/intern2 **

@maruilian11 please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/danielgindi/jquery-backstretch/
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/jquery.backstretch
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 13 
   - Count of watchers: 62
   - Count of forks: 904 (from upstream, this is a forked library)
 * NPM download stats (optional):
   - Downloads in the last day: 31
   - Downloads in the last week: 157
   - Downloads in the last month: 1050

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

autoupdate config added, close #7296, cc @danielgindi

### CODE REVIEW

File: ajax/libs/danielgindi-jquery-backstretch/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "background",
+    "photo",
+    "stretch"
+  ],
+  "npmName": "jquery.backstretch",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "jquery.backstretch*"
```
Comment (maruilian11) [inline ajax/libs/danielgindi-jquery-backstretch/package.json]: do we change to use `library*` rather than `library*.js`?

Comment (PeterDaveHello) [inline ajax/libs/danielgindi-jquery-backstretch/package.json]: I have no idea what are you talking about ...

Comment (maruilian11) [inline ajax/libs/danielgindi-jquery-backstretch/package.json]: why not use `jquery.backstretch*.js`?

Comment (PeterDaveHello) [inline ajax/libs/danielgindi-jquery-backstretch/package.json]: It's fine.

**Pull request merged**